python/cugraph-pyg/README.md

```diff
@@ -0,0 +1,5 @@
+# cugraph-pyg
+
+## Description
+
+cuGraph-PyG supports PyTorch Geometric (PyG) and implements PyG's GraphStore, FeatureStore, Loader, and Sampler interfaces.
```

Comment:
All the README-related changes in this PR are in response to `twine check --strict` errors like this:

```text
Checking 
final_dist/pylibwholegraph_cu11-24.12.0a14-cp310-cp310-manylinux_2_28_aarch64.wh
l: FAILED
ERROR    `long_description` has syntax errors in markup and would not be        
         rendered on PyPI.                                                      
         No content rendered from RST source.                                   
WARNING  `long_description_content_type` missing. defaulting to `text/x-rst`.   
Error: Process completed with exit code 1.
```

([build link](https://github.com/rapidsai/cugraph-gnn/actions/runs/11821049054/job/32935010607#step:13:725))

Proposing these checks because if / when these packages are ever uploaded to PyPI, the READMEs will be come the project pages in the PyPI UI.

I created these super-minimal READMEs by copying from https://github.com/rapidsai/cugraph-gnn/blob/branch-24.12/README.md.

@alexbarghi-nv let me know what you'd like to do:

* keep these changes
* just do not run `twine check --strict` in CI
* add some other README content

Reply:
These sound fine to me!

Reply:
The minimal READMEs are good.